mlflow/onnx.py

```diff
@@ -170,10 +170,17 @@ def predict(self, dataframe):
             }
         else:
             feed_dict = {self.inputs[0][0]: dataframe.values}
-
         predicted = self.rt.run(self.output_names, feed_dict)
-        return pd.DataFrame.from_dict(
-            {c: p.reshape(-1) for (c, p) in zip(self.output_names, predicted)})
+
+        def format_output(data):
+            # Output can be list and it should be converted to a numpy array
+            # https://github.com/mlflow/mlflow/issues/2499
+            if isinstance(data, list):
+                data = np.asarray(data)
```

Comment:
Can we attempt to do this unconditionally? There are other types (e.g., tuple) that we may want to convert to numpy arrays before attempting to reshape. I've confirmed that invoking `np.asarray()` on a numpy array correctly yields a numpy array:
```
In [1]: import numpy as np

In [2]: np.asarray(np.array([1,2,3]))
Out[2]: array([1, 2, 3])
```

Reply:
Done